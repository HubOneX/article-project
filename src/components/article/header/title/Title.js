const Title = ({ text }) => {
  return <h1 className="article-title">{text}</h1>;
};

export default Title;
