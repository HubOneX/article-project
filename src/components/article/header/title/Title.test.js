import renderer from "react-test-renderer";
import Title from "./Title";

describe("Title", () => {
  it("should match the snapshot", () => {
    const title = renderer.create(<Title text="This is a text!" />);

    expect(title).toMatchSnapshot();
  });
});
