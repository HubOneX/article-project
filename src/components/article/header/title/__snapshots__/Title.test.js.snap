// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should match the snapshot 1`] = `
<h1
  className="article-title"
>
  This is a text!
</h1>
`;
